pydarn/io/dmap.py

```diff
@@ -893,32 +902,32 @@ def __str__(self):
     # HONEY BADGER method: Because dmap just don't care
```

Comment:
Cute, but more description needed.  Why doesn't DMap care?  Is it truly a honey badger?

Reply:
Called an easter egg in code ;p Thus not in docstrings.